package.json

```diff
@@ -16,6 +16,7 @@
     "babel-preset-es2015": "~6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-register": "^6.9.0",
+    "compression-webpack-plugin": "~0.3.1",
```

Comment:
yes, I opened https://github.com/theforeman/foreman-packaging/pull/1453 for that